src/main/java/com/artipie/vertx/VertxSliceServer.java

```diff
@@ -122,29 +122,31 @@ public void close() {
      * @return The request handler.
      */
     private Handler<HttpServerRequest> proxyHandler() {
-        return (HttpServerRequest req) -> this.served.response(
-            new RequestLine(
-                req.rawMethod(),
-                req.uri(),
-                req.version().toString()
-            ).toString(),
-            req.headers(),
-            FlowAdapters.toFlowPublisher(
-                req.toFlowable().map(buffer -> ByteBuffer.wrap(buffer.getBytes()))
-            )
-        ).send(
-            (code, headers, body) -> {
-                final HttpServerResponse response = req.response().setStatusCode(code);
-                for (final Map.Entry<String, String> header : headers) {
-                    response.putHeader(header.getKey(), header.getValue());
+        return (HttpServerRequest req) -> {
+            this.served.response(
+                new RequestLine(
```

Comment:
@Sammers21 why do we need so many newlines here? It can be `new RequestLine(...).tostring()`